api/v1beta1/flinkcluster_types.go

```diff
@@ -411,6 +411,10 @@ type FlinkClusterSpec struct {
 	// Flink image spec for the cluster's components.
 	Image ImageSpec `json:"image"`
 
+	// Specifies serviceAccount name assigned to pods
+	// if empty, will use default one in namespace
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```

Comment:
Use `*string` as the type to make it optional.

Reply:
fixed